pkg/sentry/platform/ptrace/subprocess_linux.go

```diff
@@ -128,7 +128,6 @@ func attachedThread(flags uintptr, defaultAction linux.BPFAction) (*thread, erro
 		{
 			Rules: seccomp.SyscallRules{
 				syscall.SYS_GETTIMEOFDAY: {},
-				syscall.SYS_TIME:         {},
```

Comment:
Does arm64 have the vsyscall support?

Reply:
no, only gettimeofday!  Please refer to vdso/vdso_arm64.lds for more details.

Reply:
vsyscall support for getcpu is also not available on arm64. I think we need move the getcpu syscall to the arch-specific file too! Will update it soon!

Reply:
Done!

Reply:
Are you sure that there is a vsyscall call for gettimeofday? I think vsyscall was replaced by vdso before the arm64 support was added to the kernel.

Reply:
seccomp doesn't filter vdso calls.

Reply:
oh, I mixed the vdso and vsyscall concept! there is no vsyscall support for gettimeofday on arm64.

Reply:
If seccomp doesn't filter vdso calls, your current change is more reasonable. I will close this PR.